packages/parser-opts/src/index.ts

```diff
@@ -1,13 +1,13 @@
-import { gitmojiCodeRegex, gitmojiUnicodeRegex, emojiRegex } from '@gitmoji/gitmoji-regex';
+import { emojiRegex, gitmojiCodeRegex, gitmojiUnicodeRegex } from '@gitmoji/gitmoji-regex';
 
 const gitmojiCodeStr = gitmojiCodeRegex.source;
 const gitmojiUnicodeStr = gitmojiUnicodeRegex.source;
 const emojiStr = emojiRegex.source;
 
 export default {
-  // Test URL: https://regex101.com/r/gYkG99/1
+  // Test URL: https://regex101.com/r/HYr6d3/1
```

Comment:
Please add some more test examples to show the difference between before and after.

Reply:
I added these two lines at the bottom:

```
:green_heart: fix-ci(ci/cd): added release rules for all cz-emojis

:green_heart: ci/cd(config): changed config for cz-emojis
```
Here is the new URL: https://regex101.com/r/WXbn1P/1

Not sure I can show a before match in the same link.
Let me know if this is enough. 
Cheers!